src/connector/src/sink/iceberg/mod.rs

```diff
@@ -553,15 +554,52 @@ impl IcebergConfig {
                 let catalog = iceberg_catalog_rest::RestCatalog::new(config);
                 Ok(Arc::new(catalog))
             }
-            catalog_type
-                if catalog_type == "hive" || catalog_type == "jdbc" || catalog_type == "glue" =>
-            {
+            "glue" => {
+                let mut iceberg_configs = HashMap::new();
+                // glue
+                if let Some(region) = &self.region {
+                    iceberg_configs.insert(AWS_REGION_NAME.to_string(), region.clone().to_string());
+                }
+                iceberg_configs.insert(
+                    AWS_ACCESS_KEY_ID.to_string(),
+                    self.access_key.clone().to_string(),
+                );
+                iceberg_configs.insert(
+                    AWS_SECRET_ACCESS_KEY.to_string(),
+                    self.secret_key.clone().to_string(),
+                );
+                // s3
+                if let Some(region) = &self.region {
+                    iceberg_configs.insert(S3_REGION.to_string(), region.clone().to_string());
+                }
+                if let Some(endpoint) = &self.endpoint {
+                    iceberg_configs.insert(S3_ENDPOINT.to_string(), endpoint.clone().to_string());
+                }
+                iceberg_configs.insert(
+                    S3_ACCESS_KEY_ID.to_string(),
+                    self.access_key.clone().to_string(),
+                );
+                iceberg_configs.insert(
+                    S3_SECRET_ACCESS_KEY.to_string(),
+                    self.secret_key.clone().to_string(),
+                );
+                let config_builder = iceberg_catalog_glue::GlueCatalogConfig::builder()
+                    .warehouse(self.path.clone())
+                    .props(iceberg_configs);
+                let config = if let Some(uri) = self.uri.as_deref() {
+                    config_builder.uri(uri.to_string()).build()
+                } else {
+                    config_builder.build()
+                };
+                let catalog = iceberg_catalog_glue::GlueCatalog::new(config).await?;
+                Ok(Arc::new(catalog))
+            }
+            catalog_type if catalog_type == "hive" || catalog_type == "jdbc" => {
```

Comment:
Are we going to change other types too later?

Reply:
Yes, I am trying to change the `hive` catalog later by `iceberg-catalog-hms`. But the `jdbc` one can't be changed with iceberg v0.3.0